dataprovider/providers/cloud/data_provider.go

```diff
@@ -32,12 +34,17 @@ const (
 	cloudAccountNameField = "cloud.account.name"
 	cloudProviderField    = "cloud.provider"
 	cloudRegionField      = "cloud.region"
+	// TODO: update fields names when an ECS field is decided
+	cloudOrganizationIdField   = "cloud.Organization.id"
+	cloudOrganizationNameField = "cloud.Organization.name"
```

Comment:
```suggestion
	cloudOrganizationIdField   = "cloud.organization.id"
	cloudOrganizationNameField = "cloud.organization.name"
```

Reply:
Capitalized as non-final since they don't exist in ECS yet

Reply:
Makes sense, however, one of the reasons for adding those fields is to help users perform insightful queries and they would expect an ECS format.